src/server/data-access/MwBotWikibaseRepo.ts

```diff
@@ -0,0 +1,42 @@
+import WikibaseRepo from '@/server/data-access/WikibaseRepo';
+import mwbot from 'mwbot';
+import TechnicalProblem from '@/server/data-access/error/TechnicalProblem';
+import EntityNotFound from '@/server/data-access/error/EntityNotFound';
+import Entity from '@/store/entity/Entity';
```

Comment:
This `Entity` interface used here and in the following changes is the vuex store's entity module's state interface, which may or may not be the same as an entity object's json serialization. I was at some point very tempted to create an `EntityJson` (bad name?) or similar to have the separation, but was unsure about it.They have the same properties at the moment, but I find it a bit odd conceptually.